packages/gasket-plugin-docusaurus/lib/docs-view.js

```diff
@@ -0,0 +1,38 @@
+const defaultsDeep = require('lodash.defaultsdeep');
+const { writeFile, readFile } = require('fs').promises;
+const { start } = require('@docusaurus/core/lib');
```

Comment:
Maybe fine, but is this `start` function "publicly" exported somewhere so we can avoid importing from lib?

Reply:
It's similar to the existing Docsify implementation [here](https://github.com/godaddy/gasket/blob/main/packages/gasket-plugin-docsify/lib/docs-view.js#L2)

Both packages are CLI's a probably expected to be ran as such.

Reply:
👍  makes sense, just wanted to verify if you looked for a different import path.